### PR TITLE
Feat/initial setup

### LINKED ISSUE #3: Code Review Baloise Deployment Pipeline
Matthias to set up 1h slot week 45
to review the design of the [existing library](https://github.com/baloise/jenkins-shared-library-archive)

### PR DESCRIPTION
**Pull request by culmat**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Usage documentation
+
+You can either browse our example pipelines or have a look at the detailed documentation for the supported functions.
+
+
+### Shared Pipeline vars
+
+- [addJenkinsWebhook](./vars/addJenkinsWebhook.md)
```
Comment (joachimprinzbach) [inline docs/README.md]: remove the old stuff

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [getPullRequestComments](./vars/getPullRequestComments.md)
+- [getPullRequestsForBranch](./vars/getPullRequestsForBranch.md)
+- [helmLint](./vars/helmLint.md)
+- [helmPush](./vars/helmPush.md)
+- [syncApps](./vars/syncApps.md)
+
+### Design / Architecture decisions
+
+- [001-open-source-shared-jenkins-library](./decisions/001-open-source-shared-jenkins-library.md)
+- [002-usage-and-import-of-shared-library](./decisions/002-usage-and-import-of-shared-library.md)
```
Comment (joachimprinzbach) [inline docs/README.md]: remove the old stuff

File: docs/vars/greet.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# containerBuild
```
Comment (joachimprinzbach) [inline docs/vars/greet.md]: needs to be updated for greet

**Comment by joachimprinzbach** [comment]
imo this is something we are going to throw away after the PoC, right? Then it's fine for me as is.

**Comment by culmat** [comment]
> 
> 
> imo this is something we are going to throw away after the PoC, right? Then it's fine for me as is.

IMHO this is an MVP, not a PoC and we don't throw anything  away, but rather build on it.
So the foundation should be well laid.

**Pull request merged**